common/src/main/java/com/wynntils/utils/MathUtils.java

```diff
@@ -75,6 +75,10 @@ public static float magnitude(float x, float y) {
         return (float) Math.sqrt(x * x + y * y);
     }
 
+    public static float magnitude(double x, double y) {
+        return (float) Math.sqrt(x * x + y * y);
+    }
+
```

Comment:
Why? We have the float version already? I can see this added but then remove the float cast and return a double

Reply:
Oops. On a tangent, most `MathUtils` methods can be wholesale replaced with methods from `Mth`